README.md

```diff
@@ -314,15 +320,16 @@ export default Component.extend({
 Optionally you can use `ability` computed to simplify the syntax:
 ```js
 import Component from '@ember/component';
+import { inject as service } from '@ember/service';
 import { ability } from 'ember-can/computed';
 
-export default Component.extend({
-  can: service(), // inject can service
+export default class extends Component {
+  @service can; // inject can service
 
-  post: null, // received from higher template
+  post = null; // received from higher template
 
-  ability: ability('post')
-});
+  @ability('post') ability;
```

Comment:
I dont think ability is a decorator. 🤔

Reply:
@Exelord It works the same. It's a high order function. We have been using it like that in our code base.